helm-charts/dnsbl-exporter/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+annotations:
+  category: Exporter
+apiVersion: v2
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: v0.6.0
+
+dependencies:
+  - name: common
```

Comment:
@zentavr Forgot to mention this, can you explain what `bitnami-common` installs?

Reply:
I use a lot of pre-defined functions and template generators which bitnami provides. You can check the sources [here](https://github.com/bitnami/charts/tree/main/bitnami/common)

Reply:
I would really like to simplify this and have less dependency on bitnami code if possible. What do you think?

Reply:
@till that means we need to export the templates we use from that chart into our chart. Need to check their licensing if we can do that without any problems.

Reply:
They use APACHE-2.0

Reply:
@till check this: https://www.mend.io/blog/top-10-apache-license-questions-answered/